Website/DesktopModules/AuthenticationServices/DNN/Login.ascx

```diff
@@ -39,39 +40,41 @@
         </div>
     </div>
 </div>
-<script type="text/javascript">
-	/*globals jQuery, window, Sys */
-	(function ($, Sys) {
-		function setUpLogin() {
-			var actionLinks = $("a#dnn_ctr<%#ModuleId > Null.NullInteger ? ModuleId.ToString() : ""%>_Login_Login_DNN_cmdLogin");
-			actionLinks.click(function () {
-				if ($(this).hasClass("dnnDisabledAction")) {
-					return false;
-				}
+<dnn:DnnScriptBlock runat="server">
```

Comment:
Could you explain to me this control

Reply:
Hi @tpluscode , no special to this control, just include the <%=...%> block in a server control so it becomes a child control of this one but not the login control, then modify the control collection of login control won't be failed.